tests/model_hub_tests/transformation_tests/models/hf-tiny-random-vl-models-precommit

```diff
@@ -0,0 +1,2 @@
+katuni4ka/tiny-random-llava-next,https://huggingface.co/katuni4ka/tiny-random-llava-next
+katuni4ka/tiny-random-minicpmv-2_6,https://huggingface.co/katuni4ka/tiny-random-minicpmv-2_6
```

Comment:
do we actually need a separate file for these models?

Reply:
that's probably, not v1, it is VL - visual language, btw I have more modes if you need

https://huggingface.co/katuni4ka/tiny-random-llava
https://huggingface.co/katuni4ka/tiny-random-nanollava


can be added little bit later (when PRs on optimum-intel will be merged)

https://huggingface.co/katuni4ka/tiny-random-internvl2
https://huggingface.co/katuni4ka/tiny-random-phi3-vision

Reply:
@CuriousPanCake could you also add and check the new models?

Reply:
Yeah, vl in the file name stands for "visual language".
Added katuni4ka/tiny-random-nanollava & katuni4ka/tiny-random-llava

Reply:
> do we actually need a separate file for these models?
Well, I also thought about it for a while, but finally decided to go with the separate files approach. VLMs and Text-only models require different Optimum interface. Keeping them in a same file would require tracking a type of each model and different handling anyway which would easily become a mishmash so I decided to avoid it. Moreover, this looks just simpler and easier to grasp for a newcomer to see models of different types separated in different files.